src/commands.rs

```diff
@@ -75,16 +78,17 @@ fn find_device(config: &config::Config) -> anyhow::Result<nitrokey::DeviceInfo>
           .serial_number
           .map(|sn| config.serial_numbers.contains(&sn))
           .unwrap_or_default()
-    });
+    })
+    .filter(|device| config.usb_path.is_none() || config.usb_path.as_ref() == Some(&device.path));
 
   let device = iter
     .next()
     .with_context(|| format!("Nitrokey device not found{}", format_filter(config)))?;
 
   anyhow::ensure!(
     iter.next().is_none(),
-    "Multiple Nitrokey devices found{}.  Use the --model and --serial-number options to \
-    select one",
+    "Multiple Nitrokey devices found{}.  Use the --model, --serial-number and --usb-path options \
```

Comment:
I think in American English (which I try to use here) we'd use a comma before 'and' or 'or' in enumerations with more than two elements (https://en.wikipedia.org/w/index.php?title=Serial_comma&oldid=976699191).